mozilla/fx-private-relay
# Use Premium logo when the user has Premium

# New feature description

This updates the Relay logo to include "Premium" when the user has Premium.

# Screenshot (if applicable)

With Premium:

![image](https://user-images.githubusercontent.com/4251/146942836-2473e275-657b-45ca-ab2c-6bf0ebbec276.png)

Without Premium:

![image](https://user-images.githubusercontent.com/4251/146942913-5e0fa7ea-d007-4e63-b813-f6ed6b8e70fa.png)

# How to test

Sign in with an account with Premium, verify that the logo is updated. Sign in with an account without Premium, verify that the logo is unchanged.

# Checklist
